contrib/runners/python_runner/python_runner/python_runner.py

```diff
@@ -93,7 +94,7 @@ def __init__(self, runner_id, config=None, timeout=PYTHON_RUNNER_DEFAULT_ACTION_
         self._config = config
         self._timeout = timeout
         self._enable_common_pack_libs = cfg.CONF.packs.enable_common_libs or False
-        self._log_level = log_level
+        self._log_level = log_level or cfg.CONF.actionrunner.python_runner_log_level
```

Comment:
There's already a log_level in the python_runner's runner.yaml file at https://github.com/StackStorm/st2/blob/master/contrib/runners/python_runner/python_runner/runner.yaml#L16. How will this change impact the default in the runner parameter?

Reply:
You are correct - now the default parameter value in runner.yaml is a bit redundant (one in the config has priority over the one in the runner metadata file).

The reason for that is that runner.yaml is not really meant to be modified by a user / operator while config is.

To reduce the confusion, I can also remove default value from the runner.yaml in the future (will leave it there for now for backward compatibility reasons).

Reply:
I went back and look at the st2actions container.  https://github.com/StackStorm/st2/blob/master/st2actions/st2actions/container/base.py#L340 I don't even think the log_level is passed in anywhere. I don't even think there's a backward compat issue here. Anyway, I trust you'll remember to come back and remove default value from the runner.yaml.